tests/local/conftest.py

```diff
@@ -0,0 +1,19 @@
+from browserstack.local import Local
+import pytest
+from selenium.webdriver.common.by import By
+import os
+BROWSERSTACK_ACCESS_KEY = os.environ['BROWSERSTACK_ACCESS_KEY']
```

Comment:
```suggestion
BROWSERSTACK_ACCESS_KEY = os.environ['BROWSERSTACK_ACCESS_KEY'] if 'BROWSERSTACK_ACCESS_KEY' in os.environ else None
```
This will prevent error if the env variable was not found..

Reply:
Created one single conftest.py and made the change there. https://github.com/browserstack/pytest-browserstack/pull/2/commits/29aa9a24727e13c6fc580c363fc6da5fb6db1249